spring-integration-gcp/src/main/java/org/springframework/integration/gcp/pubsub/outbound/PubSubMessageHandler.java

```diff
@@ -87,4 +117,43 @@ public void setMessageConverter(MessageConverter messageConverter) {
 				"The specified message converter can't be null.");
 		this.messageConverter = messageConverter;
 	}
+
+	public boolean isSync() {
+		return this.sync;
+	}
+
+	public void setSync(boolean sync) {
+		this.sync = sync;
+	}
+
+	public Expression getPublishTimeoutExpression() {
+		return this.publishTimeoutExpression;
+	}
+
+	/**
+	 * Sets the SpEL expression to evaluate a timeout in milliseconds for a synchronous publish call
+	 * to Google Cloud Pub/Sub.
+	 *
+	 * @param publishTimeoutExpression the {@link Expression} for the publish timeout in
+	 *                                 milliseconds
+	 */
+	public void setPublishTimeoutExpression(Expression publishTimeoutExpression) {
```

Comment:
Also, please, add a variant for `String` parsing:
```
public void setPublishTimeoutExpressionString(String publishTimeoutExpression) {
		setPublishTimeoutExpression(EXPRESSION_PARSER.parseExpression(publish));
}
```
Really useful for Java config.